include/rdma/fi_prov.h

```diff
@@ -128,6 +131,17 @@ fi_param_get_bool(struct fi_provider *provider, const char *param_name, int *val
 	return fi_param_get(provider, param_name, value);
 }
 
+/* Prepend DIRECT_FN to provider specific API functions for global visibility
+ * when using fabric direct.  If the API function is static use the STATIC
+ * macro to bind symbols globally when compiling with fabric direct.
+ */
+#ifdef FI_DIRECT_ENABLED
+#define DIRECT_FN __attribute__((visibility ("default")))
+#define STATIC
+#else
+#define DIRECT_FN
+#define STATIC static
```

Comment:
Does this definition of STATIC have to be here?  Seems like it should be up to the provider whether the functions are static if FI_DIRECT is not being used.